pgmax/fg/graph.py

```diff
@@ -853,24 +854,34 @@ def tree_unflatten(cls, aux_data, children):
         return cls(**aux_data.unflatten(children))
 
 
-def BP(
-    bp_state: BPState, num_iters: int, temperature: float = 0.0
-) -> Tuple[Callable, Callable, Callable]:
+class BPContainer(NamedTuple):
+    """A container for belief propagation functions.
+
+    Arguments:
+        init: Function for creating log_potentials, ftov_msgs and evidence.
+        update: Function for updating log_potentials, ftov_msgs and evidence.
+        run_bp: Function for running belief propagation for num_iters with a damping_factor.
+        get_bp_state: Function to reconstruct the BPState from BPArrays.
+        get_beliefs: Function to calculate beliefs from BPArrays.
```

Comment:
Copy the respective docstrings here so that people know how to use these? The docstrings in individual functions won't be accessible in the documentation. Only the docstrings here would be